### PR TITLE
Refactor on key binding and matching

### PR DESCRIPTION
**Pull request by semihbkgr**

### CODE REVIEW

**Comment by canack** [comment]
Awesome 🙏🏻 I'll review it tonight. 🚀

File: internal/terminal/handler/keymap.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+var keys = keyMap{
+	SwitchTabRight: teakey.NewBinding(
+		teakey.WithKeys("shift+right"),
+	),
+	SwitchTabLeft: teakey.NewBinding(
+		teakey.WithKeys("shift+left"),
+	),
+	Quit: teakey.NewBinding(
+		teakey.WithKeys("ctrl+q"),
+	),
```
Comment (canack) [inline internal/terminal/handler/keymap.go:22]: We use `ctrl+c`  to exit any program. This is breaking the generic action. Can you replace with `ctrl+c`?

**Review by canack — CHANGES_REQUESTED** [review]
Almost done ⚙️

File: internal/terminal/handler/keymap.go (hunk 1 above)
Comment (semihbkgr) [reply internal/terminal/handler/keymap.go:22]: for sure 👍

**Review by canack — APPROVED** [review]
Ready to merge 🚀

**Pull request merged**